build/d3-bubbletreemap.js

```diff
@@ -30,8 +30,10 @@
                 let contourClusterParentUncertainty = clusterParent.uncertainty/2;                                // Padding for contour (contour lies 50% outside of parent node).
                 let planckClusterParentUncertainty = node !== clusterParent ? clusterParent.uncertainty : 0;      // Padding for force based layout (contours should not cut each other, i.e. full parent contour should be taken into account).
 
+                let interClusterSpacing = clusterNodes.length === 1 ? 0 : 5; // Add constant padding for single circle in group.
```

Comment:
Why is the padding constant? Shouldn't this be set according to the global padding variable?